docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+
+  tomcat:
+    image: tomcat:9-jdk11
+    restart: always
+    ports:
+      - 8080:8080
+    volumes:
+      - ./build/libs/SpringMovies-1.0-SNAPSHOT.war:/usr/local/tomcat/webapps/spring-movies.war
```

Comment:
Using `volumes`, you will not be able to deliver your application as a docker image via Docker Hub (or any other registry) without the need to build it. It would be much better if the `.war` file was pushed inside the image.